src/interfaces/business-state-interfaces.ts

```diff
@@ -24,11 +26,19 @@ export interface AlternateNameIF {
   operatingName: string
```

Comment:
You might notice this interface has more lines. It's because of this new interface here^